src/main/java/shedar/mods/ic2/nuclearcontrol/crossmod/bigreactors/recipes/BigReactorsRecipesNew.java

```diff
@@ -3,13 +3,17 @@
 import net.minecraft.init.Items;
 import net.minecraft.item.ItemStack;
 
+import cpw.mods.fml.common.Loader;
 import ic2.api.item.IC2Items;
 import ic2.api.recipe.Recipes;
 import shedar.mods.ic2.nuclearcontrol.crossmod.bigreactors.CrossBigReactors;
 
 public class BigReactorsRecipesNew {
 
     public static void addRecipes() {
+        if (Loader.isModLoaded("dreamcraft")) {
```

Comment:
This value should be cached since it's used multiple times.

Reply:
Not really, it only calls it 2 or 3 times because it doesn't load all of the files at once, it loads one file or the other

Reply:
Anyway I suggest to cache it to maintain guidelines of code style introduced in GT (https://github.com/GTNewHorizons/GT5-Unofficial/blob/master/.github/workflows/test-forbidden-ismodloaded.yml).